src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -857,6 +857,7 @@ private function addTranslatorSection(ArrayNodeDefinition $rootNode, callable $e
                                 ->fixXmlConfig('locale')
                                 ->children()
                                     ->scalarNode('dsn')->end()
+                                    ->booleanNode('intl_icu_enabled')->defaultFalse()->end()
```

Comment:
```suggestion
                                    ->booleanNode('intl_icu')->defaultFalse()->end()
```

I think it's better to name it short.